apps/zbugs/src/pages/issue/issue-editor.tsx

```diff
@@ -0,0 +1,154 @@
+import {useEffect, useRef, useCallback, useState} from 'react';
+import TextareaAutosize from 'react-textarea-autosize';
+import {isCtrlEnter} from './is-ctrl-enter.ts';
+import {
+  MAX_ISSUE_DESCRIPTION_LENGTH,
+  MAX_ISSUE_TITLE_LENGTH,
+} from '../../limits.ts';
+
+import * as Y from 'yjs';
+import { useZero } from '../../hooks/use-zero.ts';
+import { useQuery } from '@rocicorp/zero/react';
+
+interface IssueEditorProps {
+  issue: { id: string; title: string; description: string };
+  onSave: (fields: { title: string; description: string }) => void;
+  disabled?: boolean;
+}
+
+function base64ToUint8Array(base64: string) {
+  const binaryString = atob(base64);
+  const length = binaryString.length;
+  const uint8Array = new Uint8Array(length);
+  for (let i = 0; i < length; i++) {
+    uint8Array[i] = binaryString.charCodeAt(i);
+  }
+  return uint8Array;
+}
+
+function uint8ArrayToBase64(uint8Array: Uint8Array) {
+  const binaryString = String.fromCharCode(...uint8Array);
+  return btoa(binaryString);
+}
+
+export default function IssueEditor({
+  issue,
+  onSave,
+  disabled = false,
+}: IssueEditorProps) {
+  const [title, setTitle] = useState(issue.title);
+  const [description, setDescription] = useState(issue.description);
+
+  const z = useZero();
+  const [document, documentResult] = useQuery(z.query.document.where('id', issue.id).one());
+
+  const ydocRef = useRef<Y.Doc>();
+  const yTitleRef = useRef<Y.Text>();
+  const yDescriptionRef = useRef<Y.Text>();
+
+  useEffect(() => {
+    const ydoc = new Y.Doc();
+    ydocRef.current = ydoc;
+
+    ydoc.on('update', (update, origin) => {
+      if (origin === 'ydoc') return;
+      const updateString = uint8ArrayToBase64(update);
+      z.mutate.document.applyUpdate({documentId: issue.id, update: updateString});
```

Comment:
hmm, this is unfortunate that we can't model uint8 arrays in the mutator protocol.